http-gateways/SUBDOMAIN_GATEWAY.md

```diff
@@ -0,0 +1,196 @@
+# Subdomain Gateway Specification
+
+![](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square)
+
+**Authors**:
+
+- Marcin Rataj ([@lidel](https://github.com/lidel))
+
+----
+
+**Abstract**
+
+Subdomain Gateway is an extension of  [PATH_GATEWAY.md](./PATH_GATEWAY.md) that
+enables website hosting compatible with web browsers relative pathing and
+security model of the web. Below should be read as a delta on top of that spec.
+
+Summary:
+
+- Requests carry the CID as a sub-domain in the `Host` header rather than as a URL path prefix
+  - Case-insensitive [CIDv1](https://docs.ipfs.io/concepts/glossary/#cid-v1) encoding is used in sub-domain (see [DNS label limits](#dns-label-limits))
+  - e.g. `{cidv1}.ipfs.example.org` instead of `example.org/ipfs/{cid}`
```

Comment:
nits (can be ignored): there are mention of example.org and example.net. Not sure which one should be used.